src/frontend/app/shared/components/list/list.component.html

```diff
@@ -30,7 +30,7 @@
           <!-- Sort Button & Drop down -->
           <div class="sort" [hidden]="(view$ | async) === 'table' || sortColumns.length < 1 || (isAddingOrSelecting$ | async) || (!(dataSource.isLoadingPage$ | async) && !(hasRowsOrIsFiltering$ | async))">
             <mat-form-field>
-              <mat-select matInput [(ngModel)]="headerSort.value" shouldPlaceholderFloat="false" [disabled]="(disableActions$ | async)" (selectionChange)="updateListSort($event.value, headerSort.direction)">
+              <mat-select matInput [(ngModel)]="headerSort.value" [placeholder]="headerSort.placeholder" shouldPlaceholderFloat="false" [disabled]="(disableActions$ | async)" (selectionChange)="updateListSort($event.value, headerSort.direction)">
```

Comment:
Valid point, it's not obvious what this drop down does. Not sure a placeholder is the right solution though, so we'll come up with some suggestions.

Reply:
We've had a chat, and for the short term we'd like to go with no placeholder and to move the order button from the right hand side of the drop down to the left. We looked at a number of alternatives (using the prefix directive to supply a label or icon, changing colours/positions, etc) but nothing really worked well. 
Just let us know if you'd like us to make this change.

Reply:
Thank you @richard-cox . We'll think about it, let's just leave it this way for the time being.